test/Engine/FFITarget/ClassFunctionality.cs

```diff
@@ -179,4 +180,13 @@ public override string ToString()
             throw new NotImplementedException();
         }
     }
+
+    public class MissingClass
+    {
+        public static long MissingMethod([In, Optional] object value)
```

Comment:
This mimics some existing types from Excel.Interop
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments

Reply:
I read that document, but didn't see any syntax like this `[In, Optional]` - what should I be searching for?

Reply:
This section
```
You can also declare optional parameters by using the .NET [OptionalAttribute](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.optionalattribute) class. OptionalAttribute parameters do not require a default value.
```